ui/src/common/track.ts

```diff
@@ -0,0 +1,10 @@
+import { createDockerDesktopClient } from "@docker/extension-api-client";
+
+const analyticsEvent = "eventVolumeBackupExtension";
```

Comment:
To be consistent with the name of the extension in DockerHub and GitHub:

```suggestion
const analyticsEvent = "eventVolumesBackupExtension";
```